open-banking-accelerator/components/com.wso2.openbanking.accelerator.gateway/src/main/java/com/wso2/openbanking/accelerator/gateway/executor/dcr/DCRExecutor.java

```diff
@@ -835,7 +835,7 @@ private void validateRequestSignature(String payload, OBAPIRequestContext obapiR
         String jwksEndpoint = decodedSSA.getAsString(jwksEndpointName);
         SignedJWT signedJWT = SignedJWT.parse(payload);
         String alg = signedJWT.getHeader().getAlgorithm().getName();
-        JWTUtils.validateJWTSignature(payload, jwksEndpoint, alg);
+        JWTUtils.isValidSignature(payload, jwksEndpoint, alg);
```

Comment:
Shouldn't we check the return value of isValidSignature() here?

Reply:
- I only changed the method signature following this comment  https://github.com/wso2/financial-open-banking/pull/67#discussion_r1608181077 by vimukthi ayya.
- Furthur more the method JWTUtils.isValidSignature(which is already there) returns true unless thrown a exception